e2e-tests/src/ProjectTest.js

```diff
@@ -146,21 +143,18 @@ export const flow = {
             "selectors": [
                 [
                     "aria/My New Project June 3 2022"
-                ],
-                [
-                    "#root > div > main > div.MuiContainer-root.MuiContainer-maxWidthLg.css-1oifrf6 > div > div:nth-child(1) > div > div:nth-child(2) > div > div.MuiGrid-root.MuiGrid-container.css-11ngdyu > div:nth-child(1) > div > div.jss6 > a"
                 ]
             ],
             "offsetX": 37,
             "offsetY": 11.1875
         },
         {
             "type": "navigate",
-            "url": "http://localhost:3000/dao/b32b9ebb-b4b1-4d12-9e55-2aa62178cdf8/projects",
+            "url": "https://deploy-preview-67--sporosdaoapp-dev.netlify.app/dao/b32b9ebb-b4b1-4d12-9e55-2aa62178cdf8/projects",
```

Comment:
Where is the part of the flow that creates this DAO so it is available to test? Remember that all tests should pass in CI or any local dev environment.

Reply:
It's impossible to test creating DAO via test script because we cannot mock the metamask sign in the script.

Reply:
There are two options here:
1. Populate mongodb before running a test then clean up after test run.
2. Use metamask e2e extension such as dappeteer:
   https://github.com/chainsafe/dappeteer
   https://archive.devcon.org/archive/watch/5/make-the-tests-feel-the-users-pain-e2e-testing-for-dapps/?tab=YouTube